hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/NodeStateManager.java

```diff
@@ -289,7 +292,7 @@ private void initialiseState2EventMap() {
    */
   private void initializeStateMachine() {
     stateMachine.addTransition(
-        NodeState.HEALTHY_READONLY, NodeState.HEALTHY,
+        NodeState.HEALTHY_READONLY, HEALTHY,
         NodeLifeCycleEvent.LAYOUT_MATCH);
```

Comment:
Nit: as we statically import HEALTHY_READONLY, we can leave out the qualification of NodeState here, as we do in forceNodesToHealthyReadOnly

Reply:
yup, will change.